spec/controllers/annotation_layers_controller_spec.rb

```diff
@@ -3,41 +3,46 @@
 
 include Warden::Test::Helpers
 Warden.test_mode!
-include Devise::TestHelpers
 
 RSpec.describe AnnotationLayersController, type: :controller do
 
     context 'when Post is called' do
       describe 'POST annotation json' do
         before(:each) do
-          @annoLayerString ='{"label": "Layer 2", "motivation": "yale:transcribing", "license": "http://creativecommons.org/licenses/by/4.0/", "@type": "sc:Layer", "@context": "http://iiif.io/api/presentation/2/context.json", "otherContent": "http://localhost:5000/lists/1"}'
+          @layer = {
+            '@context' => 'http://iiif.io/api/presentation/2/context.json',
+            '@type' => 'sc:Layer',
+            'label' => 'Layer 2',
+            'otherContent' => [ 'http://localhost:5000/lists/1' ]
+          }
         end
         it 'returns a 201 ("created") response' do
-          post :create, JSON.parse(@annoLayerString), :format => 'json', :Content_Type => 'application/json', :Accept => 'application/json'
+          post_to_create(@layer)
           expect(response.status).to eq(201)
         end
 
         it 'creates a new Layer' do
-          expect { post :create, JSON.parse(@annoLayerString) }.to change(AnnotationLayer, :count).by(1)
+          expect { post_to_create(@layer) }.to change(AnnotationLayer, :count).by(1)
         end
 
         it 'creates an @id for the returned layer' do
-          post :create, JSON.parse(@annoLayerString)
+          post_to_create(@layer)
+
           expect(response.status).to eq(201)
           json = JSON.parse(response.body)
           expect(json['@id']).to be_truthy
         end
 
         it 'assigns the version' do
-          post :create, JSON.parse(@annoLayerString)
+          post_to_create(@layer)
           @annotationLayer = AnnotationLayer.last()
           expect(@annotationLayer['version']).to eq (1)
         end
 
         it 'does not fail validation if otherContent is nil' do
-          json = JSON.parse(@annoLayerString)
-          json['otherContent'] = nil
-          post :create, json
+          layer = @layer
```

Comment:
Isn't this line unnecessary? You're modifying layer in the line below which should change `@layer` and then it just gets undone again before the next test.

Do we need an `@layer.clone` or the equivalent?